components/admin_console/permission_schemes_settings/edit_post_time_limit_modal/edit_post_time_limit_modal.jsx

```diff
@@ -67,7 +67,7 @@ export default class EditPostTimeLimitModal extends React.PureComponent {
         if (value === Constants.ALLOW_EDIT_POST_ALWAYS) {
             this.setState({postEditTimeLimit: Constants.UNSET_POST_EDIT_TIME_LIMIT});
         } else {
-            this.setState({postEditTimeLimit: ''});
+            this.setState({postEditTimeLimit: 0});
```

Comment:
is this intended? if so, is it related to this change?

Reply:
Nope... that snug in from another change I was working on... will revert and push in a minute! Nice catch! 😬